devito/passes/clusters/temporal.py

```diff
@@ -0,0 +1,78 @@
+from devito.passes.clusters.utils import cluster_pass
```

Comment:
This comment is about the file, not this specific line (I didn't know where to put it)

I don't like the name `temporal`. Clearly I know where it comes from, but I like to stick to the idea that time is no special in Devito. What makes time so special? Let's see if we can devise a better name from answering this question

Reply:
I think skewing is dangerous, just because of the DSL side (skewing can mean symmetrizing too). But depending on the complexity (this seem short) I would think it just belongs to blocking.

Reply:
Maybe in the future should be sub-pass like addition in blocking? (This is what you mean ?)

Reply:
No I mean skewing probably belings to `blocking.py` rather than its own file

Reply:
Now that Mathias brings this up, I think I agree with him.  A separate file smells sounds like an overkill